### PR TITLE
[PLAT-470] Add additional pallets

### PR DESCRIPTION
**Pull request by DidacSF**
## Description

See https://ajunanetwork.atlassian.net/browse/PLAT-470 for details. In short:

* Added `pallet-identity`, `pallet-proxy` and `pallet-scheduler` to Bajun runtime.
*  Added `pallet-identity`, `pallet-proxy` and to solo runtime.

## Type of changes

- [ ] `build`: Changes that affect the build system or external dependencies (eg, Cargo, Docker)
- [ ] `ci`: Changes to CI configuration
- [ ] `docs`: Changes to documentation only
- [x] `feat`: Changes to add a new feature
- [ ] `fix`: Changes to fix a bug
- [ ] `refactor`: Changes that do not alter functionality
- [ ] `style`: Changes to format the code
- [ ] `test`: Changes to add missing tests or correct existing tests

## Checklist

- [x] Tests for the changes have been added
- [x] Necessary documentation is added (if appropriate)
- [x] Formatted with `cargo fmt --all`
- [x] Linted with `cargo clippy --all-features --all-targets`
- [x] Tested with `cargo test --workspace --all-features --all-targets`

### CODE REVIEW

**Comment by DidacSF** [comment]
Weights are missing from `pallet-scheduler`, when trying to run the benchmarks for it I get the following error:

```sh
2022-09-23 11:51:15 assembling new collators for new session 0 at #0    
2022-09-23 11:51:15 assembling new collators for new session 1 at #0    
2022-09-23 11:51:17 panicked at 'called `Result::unwrap()` on an `Err` value: ()', /home/ajuna/.cargo/git/checkouts/substrate-7e08433d4c370a21/563f482/frame/scheduler/src/benchmarking.rs:53:77    
Error: Input("Error executing and verifying runtime benchmark: Execution aborted due to trap: wasm trap: wasm `unreachable` instruction executed\nWASM backtrace:

    0: 0x2da05d - <unknown>!rust_begin_unwind
    1: 0x29d3da - <unknown>!core::panicking::panic_fmt::h1b465604beffa55a
    2: 0x29d743 - <unknown>!core::result::unwrap_failed::h83c7588b7f3a61fc
    3: 0x247f52 - <unknown>!pallet_scheduler::benchmarking::fill_schedule::h67cdf4a3183bcbfc
    4: 0x246930 - <unknown>!<pallet_scheduler::benchmarking::SelectedBenchmark as frame_benchmarking::utils::BenchmarkingSetup<T>>::instance::hd49f7cae0f58c82e
    5: 0x10c079 - <unknown>!pallet_scheduler::benchmarking::<impl frame_benchmarking::utils::Benchmarking for pallet_scheduler::pallet::Pallet<T>>::run_benchmark::h41461390be757563
    6: 0x20ab69 - <unknown>!<bajun_runtime::Runtime as frame_benchmarking::utils::runtime_decl_for_Benchmark::Benchmark<sp_runtime::generic::block::Block<sp_runtime::generic::header::Header<u32,sp_runtime::traits::BlakeTwo256>,sp_runtime::generic::unchecked_extrinsic::UncheckedExtrinsic<sp_runtime::multiaddress::MultiAddress<<<sp_runtime::MultiSignature as sp_runtime::traits::Verify>::Signer as sp_runtime::traits::IdentifyAccount>::AccountId,()>,bajun_runtime::Call,sp_runtime::MultiSignature,(frame_system::extensions::check_non_zero_sender::CheckNonZeroSender<bajun_runtime::Runtime>,frame_system::extensions::check_spec_version::CheckSpecVersion<bajun_runtime::Runtime>,frame_system::extensions::check_tx_version::CheckTxVersion<bajun_runtime::Runtime>,frame_system::extensions::check_genesis::CheckGenesis<bajun_runtime::Runtime>,frame_system::extensions::check_mortality::CheckMortality<bajun_runtime::Runtime>,frame_system::extensions::check_nonce::CheckNonce<bajun_runtime::Runtime>,frame_system::extensions::check_weight::CheckWeight<bajun_runtime::Runtime>,pallet_transaction_payment::ChargeTransactionPayment<bajun_runtime::Runtime>)>>>>::dispatch_benchmark::h4c6090ef66af98e8
    7: 0x128d69 - <unknown>!Benchmark_dispatch_benchmark\n")
```

It seems related to the `Preimage` section, not sure why it fails though

File: runtime/bajun/src/lib.rs [hunk 1]
```diff
@@ -590,6 +594,74 @@ impl pallet_ajuna_awesome_avatars::Config for Runtime {
...
+	type Event = Event;
+	type Origin = Origin;
+	type PalletsOrigin = OriginCaller;
+	type Call = Call;
+	type MaximumWeight = MaximumSchedulerWeight;
+	type ScheduleOrigin = EnsureRoot<AccountId>;
+	type OriginPrivilegeCmp = frame_support::traits::EqualPrivilegeOnly;
+	type MaxScheduledPerBlock = MaxScheduledPerBlock;
+	type WeightInfo = ();
+	type PreimageProvider = ();
```
Comment (cowboy-bebug) [inline runtime/bajun/src/lib.rs:661]: Could you also add `pallet_preimage` and use it for PreimageProvider?
```suggestion
	type PreimageProvider = Preimage;
```
I'm not sure why these pallets are decoupled :smile:

**Review by cowboy-bebug — COMMENTED** [review]
Could you also:
- remove these pallets from solo because we don't want to maintain solo runtime yet
- add pallets in: https://github.com/ajuna-network/Ajuna/blob/5477908ad43919657f3f4904dc469f2d1d8cb95c/.maintain/weights.sh#L7-L19

File: runtime/bajun/src/lib.rs (hunk 1 above)
Comment (DidacSF) [reply runtime/bajun/src/lib.rs]: Yeah, strange decision

**Review by cowboy-bebug — APPROVED** [review]
LGTM!

**Pull request merged**